docs/api-docs/store-management/shipping/shipping-provider-api.md

```diff
@@ -31,32 +30,80 @@ Some use cases for the Shipping Provider API are:
+
+<!--
+    title: #### Single-Carrier Quote Example
+
+    data: //raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/shipping-provider-single-carrier-ui.png
+-->
 
+#### Single-Carrier Quote Example
 
-To get your app ID, create an app in [Developer Tools](https://devtools.bigcommerce.com/), and fill out the information on [Step 3 Technical](https://developer.bigcommerce.com/api-docs/apps/guide/publishing#add-technical-information). In the URL, the app will have a unique ID. Send the unique ID in exchange for a carrier ID to test the app.
+![#### Single-Carrier Quote Example](//raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/shipping-provider-single-carrier-ui.png "#### Single-Carrier Quote Example
+")
+
+<!--
+    title: #### Multi-Carrier Quote Example
+
+    data: //raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/shipping-provider-multi-carrier-ui.png
+-->
+
+#### Multi-Carrier Quote Example
+
+![#### Multi-Carrier Quote Example](//raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/shipping-provider-multi-carrier-ui.png "#### Multi-Carrier Quote Example
+")
+
+## Sign up
+
+When your app is complete, it will be listed in our Carrier Registry, so that your shipping rates are available for merchants and shoppers to use. Register your app and get a carrier ID to get started.
+
+### Register your app
+
+We need your app ID to generate a carrier ID for your shipping service. To get your app ID, [create a draft app](https://developer.bigcommerce.com/api-docs/apps/guide/development#registering-a-draft-app) in [Developer Tools](https://devtools.bigcommerce.com/), and fill in the information requested on the [Step 3: Technical tab](https://developer.bigcommerce.com/api-docs/apps/guide/publishing#add-technical-information). After you save the app, the developer tools control panel will navigate to a URL that includes your app's unique ID. 
 
 ![#### App ID](//s3.amazonaws.com/user-content.stoplight.io/6012/1552664114224 "#### App ID")
 
+### Request a carrier ID
+
+To get your app set up and request a carrier ID, send an email to
+<a href="mailto:shippingproviderapi@bigcommerce.com">shippingproviderapi@bigcommerce.com</a>.
+
+Please include the following information:
+
+- Your name
+- Your partner ID
+- Your app's ID (see the following)
+- Your email
+- A description of the app
+- [Your service URLs](#your-service-urls) 
+- [Whether you prefer single-carrier or multi-carrier status](#single-carrier-vs-multi-carrier-apps)
+
 ## Before development
 
 BigCommerce will send requests to your server to get information back about shipping quotes, credential validation, and configuration.

```

Comment:
```suggestion
BigCommerce sends requests to your server to get information back about shipping quotes, credential validation, and configuration.

```

Reply:
revised and added to.  this sentence is a propos of nothing.  I added some transitional language.